pycbc/fft/fftw.py

```diff
@@ -323,14 +325,14 @@ def fft(invec, outvec, prec, itype, otype):
                             get_measure_level(),(check_aligned(invec.data) and check_aligned(outvec.data)),
                    _scheme.mgr.state.num_threads, (invec.ptr == outvec.ptr))
     execute(theplan, invec, outvec)
-    destroy(theplan)
+    #destroy(theplan)
```

Comment:
I don't think we can ignore the deletion. I think it would be necessary to see if one can update the __del__ method to get calling this worked into when the plan is removed from the cache naturally.